uk_bin_collection/uk_bin_collection/councils/LondonBoroughRedbridge.py

```
@@ -71,82 +71,80 @@ def parse_data(self, page: str, **kwargs) -> dict:
+                            # Parse the date string (format: "04 February 2026 ")
+                            date_string = f"{collection_date} {collection_month.strip()}"
+
+                            try:
+                                # Convert the date string to a datetime object
+                                collection_date_obj = datetime.strptime(
+                                    date_string, "%d %B %Y"
+                                )
+
+                                # Format the date
                                 formatted_date = collection_date_obj.strftime(
                                     date_format
```

Ruff (0.14.14)

[error] `date_format` may be undefined, or defined from star imports

(F405)